packages/http-client-java/generator/http-client-generator-core/src/main/java/com/microsoft/typespec/http/client/generator/core/model/clientmodel/ClassType.java

```diff
@@ -451,7 +451,9 @@ private static ClassType.Builder getClassTypeBuilder(Class<?> classKey) {
     public static final ClassType INPUT_STREAM = new ClassType.Builder(false).knownClass(InputStream.class).build();
 
     public static final ClassType CONTEXT = ClassType.getClassTypeBuilder(Context.class)
-        .defaultValueExpressionConverter(epr -> "com.azure.core.util.Context.NONE")
+        .defaultValueExpressionConverter(epr -> JavaSettings.getInstance().isBranded()
+            ? "com.azure.core.util.Context.NONE"
+            : "io.clientcore.core.util.Context.none()")
```

Comment:
use TemplateUtil.contextNone()?

Reply:
Yeah, we can use that. Here it just we already need to branch on the package name (we likely use the full package name to disambiguate).